codigo/node-geolocation/geolocate.js

```diff
@@ -0,0 +1,37 @@
+const request = require('request-promise')
+const utf8 = require('utf8');
+require('dotenv').config()
+let apiKey = process.env.GEOCODING_API_KEY
+
+
+let geolocate = async (address) => {
+  let curatedGeolocation = {}
```

Comment:
Better naming. curatedGeolocationResponse?

Reply:
Too long. I think is specific enough.